monolith/database.py

```diff
@@ -163,4 +163,15 @@ def is_date(string):
         return True
 
     except ValueError:
-        return False
\ No newline at end of file
+        return False
+
+def get_suggested_stories(user_id):
+    lastUsedThemes= [story.theme for story in db.session.query(Story).filter(Story.author_id == user_id).distinct()]
```

Comment:
Should be 

lastUsedThemes = Story.query.with_entities(Story.theme).distinct()

Reply:
Not working, we need to transform the query into a list in this way. Otherwise the successive comparation will not work.

Reply:
Usually is not a good idea fetch all data and discard most of it in the application.
This is negligible because SqlLite is just storage, otherwise we are moving computing power from database to business application.

But, if it's working no problem. Let's move on! 

Approved!